### DISCUSSION TITLE
22/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#170] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer clone nuevamente el repositorio porque tuve problemas con la aplicación android.
Hoy tengo reunión en el trabajo espero avanzar con las pantallas
Tiempo tiempo tiempo ⏱️

**Comment by fmdcba** [comment]
Hoy estuve leyendo el material de tecnología, buscando más información y continué avanzando con el proyecto de la materia, ya que quedamos muy atrasados en cuanto a contenido e investigación.
